src/pybind/mgr/mgr_module.py

```diff
@@ -1722,7 +1722,7 @@ def tell_command(self, daemon_type: str, daemon_id: str, cmd_dict: dict, inbuf:
         """
         t1 = time.time()
         result = CommandResult()
-        self.send_command(result, daemon_type, daemon_id, json.dumps(cmd_dict), "", inbuf)
+        self.send_command(result, daemon_type, daemon_id, json.dumps(cmd_dict), "", inbuf, one_shot=one_shot)
```

Comment:
well, it's simply a kw-only arg and that should go at the end. Why would you want it to go before the inbuf?
I prefer this new arg to be kw-only because boolean in the middle is very hard to understand if your IDE doesn't give you inline hints.
I'll update the commit message once we settle on this here in this thread

Reply:
> well, it's simply a kw-only arg and that should go at the end.

Ah, yes.

> Why would you want it to go before the inbuf?

Unless I'm missing a spot, all the commands are sent through `MgrModule.send_command` which always adds the `oneshot` kwarg. Why not just make it a positional?

> I prefer this new arg to be kw-only because boolean in the middle is very hard to understand if your IDE doesn't give you inline hints. I'll update the commit message once we settle on this here in this thread

You could keep the kwarg to `MgrModule.send_command` and make `_ceph_send_command` pass a positional. Or, only pass the kwarg to `_ceph_send_command` if `oneshot` is `not None` (a reasonable default). To me, it seemed weird to pass a kwarg specific to the CephFS Client for commands to other entities.

Reply:
I'm going to resolve this. It's small potatoes.